codetoanalyze/erlang/annotreach/reach.erl, test_source2_Bad/0, 0, CHECKERS_ANNOTATION_REACHABILITY_ERROR, no_bucket, ERROR, [Method reach:test_source2_Bad, marked as source @Source,calls reach:sink,reach:sink defined here, marked as sink @Sink]
codetoanalyze/erlang/annotreach/reach.erl, test_source4_Bad/0, 0, CHECKERS_ANNOTATION_REACHABILITY_ERROR, no_bucket, ERROR, [Method reach:test_source4_Bad, marked as source @Source,calls reach:not_sani_tizer,reach:not_sani_tizer defined here,calls reach:sink,reach:sink defined here, marked as sink @Sink]
